syncer/syncer_test.go

```diff
@@ -947,12 +947,12 @@ func (s *testSyncerSuite) TestGeneratedColumn(c *C) {
 					c.Assert(args[0], DeepEquals, testCase.args[idx])
 				case replication.UPDATE_ROWS_EVENTv0, replication.UPDATE_ROWS_EVENTv1, replication.UPDATE_ROWS_EVENTv2:
 					// test with sql_mode = false only
-					sqls, _, args, err = syncer.genUpdateSQLs(param, nil, nil)
+					sqls, _, args, err = syncer.genUpdateSQLs(tcontext.Background(), param, nil, nil)
```

Comment:
any cases the context can be different from `tcontext.Background()` and need to pass from outside? if it's not, it's not necessary to pass around,  as it's used only in `GetToIndexInfo`

Reply:
That is just for test.